app-containers/lxd/lxd-5.17.ebuild

```diff
@@ -0,0 +1,185 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit bash-completion-r1 go-module linux-info optfeature systemd verify-sig
+
+DESCRIPTION="Modern, secure and powerful system container and virtual machine manager"
+HOMEPAGE="https://ubuntu.com/lxd https://github.com/canonical/lxd"
+SRC_URI="https://github.com/canonical/lxd/releases/download/${P}/${P}.tar.gz"
+
```

Comment:
```suggestion
SRC_URI="https://github.com/canonical/lxd/releases/download/${P}/${P}.tar.gz
    verify-sig? ( https://github.com/canonical/lxd/releases/download/${P}/${P}.tar.gz.asc )"
```

Reply:
shouldn't it be now dependent on sec-keys/openpgp-keys-canonical? Like `verify-sig? ( sec-keys/openpgp-keys-canonical )`